ethereum_history_api/circuits/lib/src/verifiers/receipt_test.nr

```diff
@@ -70,3 +73,54 @@ fn test_assert_receipt_proof_fail_invalid_proof_depth() {
 
     assert_receipt_proof(proof_invalid_depth, receipts_root);
 }
+
+#[test]
+fn test_verify_receipt_success() {
+    verify_receipt(block_number, tx_idx, tx_type, receipt, proof, receipts_root);
+}
+
+#[test(should_fail_with="key does not match rlp-encoded transaction index")]
```

Comment:
We don't need invalid block_number test because `block_number` comes from `get_header` and is already verified. We need to verify values that comes from user or oracle.